### PR TITLE
Peak height mass trace

### PR DESCRIPTION
**Pull request by poshul**
Add the ability to quantify mass traces based on the highest peak. Addresses https://github.com/OpenMS/OpenMS/issues/4371

### CODE REVIEW

**Comment by timosachsenberg** [comment]
Hi looks good so far. Can you check if / how this integrates into FEatureFinderMetabo. Having a topp test for FFM would be great

**Comment by poshul** [comment]
TOPP test added.

File: src/openms/source/KERNEL/MassTrace.cpp [hunk 1]
```diff
@@ -421,6 +423,8 @@ namespace OpenMS
             return computeFwhmArea();
           case MT_QUANT_MEDIAN:
             return computeMedianIntensity_();
+          case MT_QUANT_HEIGHT:
+            return getMaxIntensity(false);
```
Comment (oliveralka) [inline src/openms/source/KERNEL/MassTrace.cpp]: Should that not work with non_smoothed data as well?

**Comment by poshul** [comment]
rebuild jenkins

**Comment by timosachsenberg** [comment]
Thanks! looks good

**Pull request merged**